client/cmd/bisonw-desktop/go.mod

```diff
@@ -1,6 +1,8 @@
 module decred.org/dcrdex/client/cmd/bisonw-desktop
 
-go 1.19
+go 1.21
+
+toolchain go1.23.0
```

Comment:
You really shouldn't set the toolchain as it will **require** everyone to build with Go 1.23 which goes against supporting Go 1.22.

Reply:
Well, unless you're being forced to by some dep, which would also showcase why you shouldn't have a toolchain directive unless you _actually_ need it.

Reply:
I have no idea why this is here. All I did was `go get` new deps and `go mod tidy`. No issues after removing them though, and `go mod tidy` doesn't seem to be inserting them. Not sure what happened.

Reply:
Yeah, I noticed that `go get` adds it now whenever it updates the required Go version (the `go` directive).  

https://tip.golang.org/doc/toolchain